src/Integrations/Clearbit/SettingsClearbit.php

```diff
@@ -0,0 +1,456 @@
+				'introTitleSize' => 'small',
+				// phpcs:ignore WordPress.WP.I18n.NoHtmlWrappedStrings
+				'introSubtitle' => __('<ol>
+						<li>Log in to your Clearbit Account.</li>
+						<li>Then click on the <strong><a target="_blank" href="https://dashboard.clearbit.com/api">API</a></strong> in the sidebar.</li>
+						<li>Copy the secret API key into the field below or use the global constant</li>
+					</ol>', 'eightshift-forms'),
+			],
+			[
+				'component' => 'divider',
+			],
+			[
+				'component' => 'checkboxes',
+				'checkboxesFieldLabel' => '',
+				'checkboxesName' => $this->getSettingsName(self::SETTINGS_CLEARBIT_USE_KEY),
+				'checkboxesId' => $this->getSettingsName(self::SETTINGS_CLEARBIT_USE_KEY),
+				'checkboxesIsRequired' => true,
+				'checkboxesContent' => [
+					[
+						'component' => 'checkbox',
+						'checkboxLabel' => __('Use Clearbit', 'eightshift-forms'),
+						'checkboxIsChecked' => $this->isCheckboxOptionChecked(self::SETTINGS_CLEARBIT_USE_KEY, self::SETTINGS_CLEARBIT_USE_KEY),
+						'checkboxValue' => self::SETTINGS_CLEARBIT_USE_KEY,
+						'checkboxSingleSubmit' => true,
+					]
+				]
+			],
+		];
+
+		$outputApi = [];
+
+		if ($isUsed) {
+			$apiKey = Variables::getApiKeyClearbit();
+
+			$outputApi = [
+				[
+					'component' => 'input',
+					'inputName' => $this->getSettingsName(self::SETTINGS_CLEARBIT_API_KEY_KEY),
+					'inputId' => $this->getSettingsName(self::SETTINGS_CLEARBIT_API_KEY_KEY),
+					'inputFieldLabel' => __('API key', 'eightshift-forms'),
+					'inputFieldHelp' => __('Can also be provided via a global variable.', 'eightshift-forms'),
+					'inputType' => 'password',
+					'inputIsRequired' => true,
+					'inputValue' => !empty($apiKey) ? 'xxxxxxxxxxxxxxxx' : $this->getOptionValue(self::SETTINGS_CLEARBIT_API_KEY_KEY),
+					'inputIsDisabled' => !empty($apiKey),
+				],
+			];
+		}
+
+		$isValid = self::isSettingsGlobalValid();
+		$outputUsage = [];
+
+		if ($isValid) {
+			$outputUsage = [
+				[
+					'component' => 'divider',
+				],
+				[
+					'component' => 'checkboxes',
+					'checkboxesFieldLabel' => __('Available fields', 'eightshift-forms'),
+					'checkboxesFieldHelp' => __('Select fields that you want to use in your forms.', 'eightshift-forms'),
+					'checkboxesName' => $this->getSettingsName(self::SETTINGS_CLEARBIT_AVAILABLE_KEYS_KEY),
+					'checkboxesId' => $this->getSettingsName(self::SETTINGS_CLEARBIT_AVAILABLE_KEYS_KEY),
+					'checkboxesIsRequired' => true,
+					'checkboxesContent' => array_map(
+						function ($item) {
+							return [
+								'component' => 'checkbox',
+								'checkboxLabel' => $item,
+								'checkboxIsChecked' => $this->isCheckboxOptionChecked($item, self::SETTINGS_CLEARBIT_AVAILABLE_KEYS_KEY),
+								'checkboxValue' => $item,
+							];
+						},
+						$this->clearbitClient->getParams()
+					),
+				],
+			];
+		}
+
+		return array_merge(
+			$output,
+			$outputApi,
+			$outputUsage
+		);
```

Comment:
Would `$output + $outputApi + $outputUsage` return the same array?

Reply:
Not sure but I would like to leave it as is now

Reply:
Sure, can you test this afterward?